packages/php-wasm/web-service-worker/src/worker-caching.ts

```diff
@@ -0,0 +1,86 @@
+// import { logger } from '@php-wasm/logger';
+import { isURLScoped } from '@php-wasm/scopes';
+
+const wpCacheKey = 'playground-cache';
+
+const isCacheFull = async () => {
+	if (navigator.storage && navigator.storage.estimate) {
+		const estimatedStorage = await navigator.storage.estimate();
+		if (estimatedStorage.usage && estimatedStorage.quota) {
+			return estimatedStorage.usage >= estimatedStorage.quota;
+		}
+	}
+	return false;
+};
+
+export const addCache = async (key: string, response: Response) => {
+	if (await isCacheFull()) {
```

Comment:
* Would the cache ever be full? E.g. say we have 150KB left and we're trying to write a 200KB file. The operation would fail even though the cache isn't full yet.
* Is this check any different from trying to write and throwing when that fails?

Reply:
You are correct, this doesn't make any sense. I removed the full cache check in [5e4d336](https://github.com/WordPress/wordpress-playground/pull/1535/commits/5e4d3361f6cfdb1a8b3df185e473d0ab18f961fb).